server-tests/methods.tests.js

```diff
@@ -0,0 +1,429 @@
+      params.message = longMessage;
+
+      Meteor.call("message.build", params, function(error, response) {
+        setTimeout(function() {
+          chai.assert.isUndefined(response);
+          chai.assert.equal(error.error, "validation-error");
+          done();
+        })
+      });
+    });
+
+    it("message.build throws exception when parameters are wrong type", function (done) {
+      params.message = 123123;
+
+      Meteor.call("message.build", params, function(error, response) {
+        setTimeout(function() {
+          chai.assert.isUndefined(response);
+          chai.assert.equal(error.errorType, "Match.Error");
+          done();
+        })
+      });
+    });
+
+    it("message.build throws exception when message text is empty", function (done) {
+      params.message = "";
+
+      Meteor.call("message.build", params, function(error, response) {
+        setTimeout(function() {
+          chai.assert.equal(error.error, "validation-error");
+          done();
+        })
+      });
+    });
+
+    it("message.build throws error when user is not part of room ", function (done) {
+      Chatter.UserRoom.remove({userId: user._id, roomId: params.roomId });
+      params.message = "test message";
+
+      Meteor.call("message.build", params, function(error, response) {
+        setTimeout(function() {
+          chai.assert.isUndefined(response);
+          chai.assert.equal(error.error, "user-not-in-room");
+          done();
+        })
+      });
+    });
+  });
+
+  describe("room methods", function (done) {
+    const roomAttributes = {
+      name: "test room",
+      description: "test description"
+    };
+
+    let params;
+    let room;
+    let user;
+
+
+    before(function() {
+      params = {};
+      const meteorUser = Meteor.users.findOne();
+      const userId = Chatter.addUser(meteorUser._id, "admin");
+
+      roomId = Chatter.Room.insert(roomAttributes);
+      room = Chatter.Room.findOne({_id: roomId});
+      user = Chatter.User.findOne({_id: userId});
+    });
+
+    describe("room.build", function (done) {
+      it("throws an error when required parameters are missing", function (done) {
+        const params = {};
+
+        Meteor.call("room.build", params, function(error, response) {
+          setTimeout(function() {
```

Comment:
These are a the two references I found, so it does seem like it is being worked on.
I will create the wrapper function though!
https://github.com/mochajs/mocha/issues/1128
https://github.com/mad-eye/meteor-mocha-web/issues/78